### PR TITLE
Add a Flask server for NL

### PR DESCRIPTION
**Pull request by shifucun**
This will be another container deployed together with website and mixer in the same Pod.

All the NL related code and endpoints should be moved to it.

### CODE REVIEW

File: server/services/datacommons.py [hunk 1]
```diff
@@ -327,6 +327,13 @@ def get_event_collection_date(event_type, affected_place):
   })
 
 
+def nl_helloworld():
+  """fetch nl hello world data.
+  """
+  url = cfg.NL_ROOT
```
Comment (shifucun) [inline server/services/datacommons.py:333]: @Fructokinase what would be a good way to put this in discovery.py? Also I wonder if we should complete the work to have mixer service (and now with nl service)

File: nl_server/routes.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# limitations under the License.
+
+import json
+
+from flask import Blueprint
+
+bp = Blueprint('main', __name__, url_prefix='/')
+
+
+@bp.route('/')
```
Comment (shifucun) [inline nl_server/routes.py:22]: @jehangiramjad You can update this function, rename this file or add new files for the NL endpoints

Comment (jehangiramjad) [reply nl_server/routes.py:22]: got it, thanks!

File: nl_server/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Flask==1.1.4
```
Comment (jehangiramjad) [inline nl_server/requirements.txt:1]: We will certainly need this (I can also add them later):

torch==1.12.1
pandas==1.3.5
spacy==3.4.4
en_core_web_md @ https://github.com/explosion/spacy-models/releases/download/en_core_web_md-3.4.1/en_core_web_md-3.4.1-py3-none-any.whl

**Review by jehangiramjad — APPROVED** [review]
Many thanks! I'll let @Fructokinase reply to the other comment

File: nl_server/requirements.txt (hunk 3 above)
Comment (shifucun) [reply nl_server/requirements.txt:1]: ok, sg, we can add those as you migrate them

File: nl_server/routes.py (hunk 2 above)
Comment (beets) [reply nl_server/routes.py:22]: my understanding is that this is strictly an API server to host bigger embeddings? in that case, we should perhaps set up with a /api endpoint.

**Review by beets — APPROVED** [review]

File: nl_server/routes.py (hunk 2 above)
Comment (shifucun) [reply nl_server/routes.py:22]: Yep. @jehangiramjad will need to update those and we can make sure they are /api/...

**Review by shifucun — COMMENTED** [review]
thanks for review!

**Pull request merged**